google/auth/external_account.py

```diff
@@ -94,6 +95,7 @@ def __init__(
                 impersonation generateAccessToken URL.
             client_id (Optional[str]): The optional client ID.
             client_secret (Optional[str]): The optional client secret.
+            token_info_url (Optional[str]): The optional STS endpoint for token info.
```

Comment:
Should this be token Introspection isntead of info?

Reply:
Done